parameters.yml.dist

```diff
@@ -28,6 +28,10 @@ database_name: "example"
 # Root directory of your project for the webserver and other purposes
 root_directory: "/vagrant/"
 
+# Synced folder type to use. Only "nfs" and "" will work. Default in drifter
```

Comment:
I would use values `nfs` and `virtualbox` to be more explicit. Also the comment mentions that only nfs and "" will work, but actually any value will work. I think we should validate this in the `Vagrantfile` (ie. make sure value is either `nfs` or `virtualbox`). Could you add that?